index.html

```diff
@@ -0,0 +1,101 @@
+        </div>
+        <div class="rightBlock">
+          <img src="./images/siarohin.jpg" alt="Aleksandr Seregin">
+        </div>
+      </div>
+      <div id="content">
+        <div id="tableGrid">
+          <div>
+            <h2>Experience</h2>
+            <p>I'am getting experience on Rolling Scopes School Courses. I am improving my skills in semantic true HTML-markup, styling CSS-sheets, JavaScript coding, and using Git. You'll see my tasks and projects on my <a href="//github.com/siarohin" title="GitHub account" target="_blank">GitHub account</a>. I also have a little experience in Ruby, and with RoR Framework.</p>
+          </div>
+          <div>
+            <h2>Skills</h2>
+              <p>I have basic knowleges of SQL, SQLite3, <strong>HTML</strong>, <strong>CSS</strong> and JS. I understand OOP. I know <span>REST</span>, ActiveRecord, irb, hash, GET/POST, Git, ungit, bootstrap, Sinatra. You'll see my progress on the <a href="//www.codecademy.com/siarohin" title="Codecademy account" target="_blank">Codecademy account</a></p>
+          </div>
+          <div>
+            <h2>Education</h2>
+            <p>I graduated from BSUIR with a degree in Radioelectronics in 2008. At the present time, I independently study web development technologies and tools. Ruby on Rails and JS are among the latest ones I’ve worked on.</p>
+          </div>
+          <div>
+            <h2>Total work</h2>
+            <p>Since 2017 I have been working as in an individual entrepreneur. My workload includes data analysis and distribution in the online shop. My activities in computer programming cover php and css formatting, web banner design, SEO and paid advertising, webpage slicing, JS script implementation, relational databases management such as MySQL.</p>
+          </div>
+          <div>
+            <h2>English</h2>
+            <p>Based on test results at Streamline, I have A2 Level in English. Since September 2018 I’ve been studying at the Pre-Intermediate Level (A2/A2+) at Streamline to improve my English skills. I’m planning to continue studying for B1 and B2 Levels.</p>
+          </div>
+        </div>
+        <div id="slider">
+          <h2>The best projects</h2>
+          <input checked type="radio" name="slider" id="slide1">
+          <input type="radio" name="slider" id="slide2">
+          <input type="radio" name="slider" id="slide3">
+          <div id="slides">
+            <div id="slideContainer">
+              <div class="inner">
+                <article>
+                  <h3>I like a tasty food</h3>
+                  <img src="./images/slides/000001.jpg" alt="I like a tasty food">
+                </article>
```

Comment:
take a look at html5 tags [figure](https://www.w3schools.com/tags/tag_figure.asp) and [figcaption](https://www.w3schools.com/tags/tag_figcaption.asp), they also can be used there